flumine/controls/tradingcontrols.py

```diff
@@ -134,18 +138,21 @@ def _validate(self, order_package):
                     )
                     continue
 
-                # per selection
+            # per selection
+            for (strategy, lookup), orders in package_orders_by_strategy_lookup.items():
+
                 market = self.flumine.markets.markets[order_package.market_id]
-                current_selection_exposure = market.blotter.selection_exposure(
-                    strategy, lookup=order.lookup
+                profit_if_win, profit_if_lose = market.blotter.selection_exposure(
```

Comment:
`profit_if_win` is this confusing as it's always zero if a back order?

Reply:
I refactored the code slightly.

Exposure is now greater than or equal to zero, and represents potential losses. If a selection is guaranteed to make a profit, then the exposure will be zero. Hopefully this communicates intent more clearly.

Reply:
Looks much better 👍 

Only concern is around `selection_exposure` returning a positive number, does that make sense to the user?

Reply:
This is a bit subjective. But to me, if it is called "exposure", then it already implies a potential loss.

If someone lost £10, they wouldn't say "I lost minus 10 pounds". They would say "I lost 10 pounds".

Also `max_selection_exposure` and `max_order_exposure` attributes are positive, so it would be inconsistent for `selection_exposure` to return negative values.

Reply:
Sounds good, I will wrap this up into a beta this morning and start testing